src/map/clif.h

```diff
@@ -1351,7 +1351,7 @@ struct clif_interface {
 	void (*pNPCMarketClosed) (int fd, struct map_session_data *sd);
 	void (*pNPCMarketPurchase) (int fd, struct map_session_data *sd);
 	/* */
-	void (*add_random_options) (unsigned char* buf, struct item* item);
+	int (*add_random_options) (struct ItemOptions *buf, const struct item *it);
```

Comment:
since we're calling them Item Options rather than Random Options, I think this method should also be renamed accordingly